src/data/events.json

```diff
@@ -188,15 +188,15 @@
     "communityLogo": "https://cdn2.allevents.in/thumbs/thumb67835596402d9.png"
   },
   {
-    "eventName": "26th Build2Learn",
+    "eventName": "27th Build2Learn",
     "eventDescription": "Welcome to build2learn - Where Innovation Meets Community! Imagine a hackathon, but with a twist!🤔 It's not just about building incredible micro products; it's about forging lifelong connections with like-minded individuals who share your passion for innovation! 🤝💡At build2learn, we believe that magic happens when engineers come together to do what they love - BUILD!, Are you someone with a side project that you've been dreaming of bringing to life? build2learn is your chance to turn that dream into reality! ⏰🌟 Are you a student eager to create something extraordinary but in need of mentors to guide you? build2learn is the launchpad you've been waiting for! 🎓🚀 Or maybe you're someone who thrives on helping others unleash their full potential? build2learn is the perfect platform to make a lasting impact! 🤗 Join us at build2learn and embark on a journey of creation, collaboration, and camaraderie! 🌟 See you there!🌆 ",
-    "eventDate": "2025-04-19",
+    "eventDate": "2025-05-17",
     "eventTime": "09:30",
-    "eventVenue": "Freshworks SP infocity",
-    "eventLink": "https://docs.google.com/forms/d/e/1FAIpQLSeIquUO7lBZt1Ca6B0FNOhLj5Z4P764gHfdFQUWXqt1B-FaiA/viewform",
+    "eventVenue": "Entrans, Millenia Business Park, Phase 2, Campus 3A, 2nd Floor, Dr. MGR Main Rd, Kandhanchavadi, Perungudi, Chennai – 600096",
+    "eventLink": "https://lu.ma/n55b1sav?utm_source=ttech",
     "location": "Chennai",
-    "communityName": "build2learn",
-    "communityLogo": "https://i.ibb.co/CBsgh43/bxOxo5O.png"
+    "communityName": "Build2Learn",
+    "communityLogo": "https://build2learn.pages.dev/images/build2learn-logo.svg"
```

Comment:
Amazing one @selvasudar bro,
love the fact that you repurposed the older event description and content for a new event. 
Thanks a lot for that. 

One tiny issue is that the community logo is hosted in a page which is not added to next.config which means that it wouldnt work. I would highly suggest you to use imgbb and use an embeddable link similar to the previous data, for it to work. 

Would love to merge it asap <3

Cheers :)